lib/kcerts/ked25519/cert.go

```diff
@@ -0,0 +1,87 @@
+package ked25519ackage
+
+import (
+	"golang.org/x/crypto/ed25519"
+	"golang.org/x/crypto/ssh"
+	"math/rand"
+)
+// Copied from https://github.com/mikesmitty/edkey/blob/master/edkey.go
+/* Writes ed25519 private keys into the new OpenSSH private key format.
```

Comment:
nit: please format this comment as per normal golang conventions, so godoc generates the documentation properly.

Eg,
```
// MarshalED25519PrivateKey writes ed25519 private keys into the new OpenSSH private key format.
```